src/storybook/pages/lean/lean.scss

```
@@ -0,0 +1,24 @@
+@use '../../../elements/core/styles' as sbb;
+
+.lean-section {
+  display: flex;
+  flex-wrap: wrap;
+  margin-block: var(--sbb-spacing-fixed-4x);
+}
+
+.lean-container {
+  width: 100%;
+  display: flex;
+  gap: var(--sbb-spacing-fixed-4x);
+  padding: var(--sbb-spacing-fixed-4x);
+  flex-wrap: wrap;
+}
+
+.lean-container-half {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  flex: 50%;
+  gap: var(--sbb-spacing-fixed-4x);
+  padding: var(--sbb-spacing-fixed-4x);
```

right and left spacing Should be aligned with header. BUT: you would need now to fix the gap. generally I recommend to use CSS grid combined with a media query.
```suggestion
  padding-block: var(--sbb-spacing-fixed-4x);
```

---

I used the `sbb-grid-expanded` class, but not sure it's the correct choice

---

but then you could just set the grid-area / grid-column?